### PR TITLE
Target patient count and duration update on UI

### PR DESCRIPTION
**Pull request by sylvr3**
User can now add a new trial and the target number of patients will reflect their user input and be displayed on the dashboard
Removed duration from description
Removed target number of patients random number calculation

### CODE REVIEW

File: presenter/dashboard.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ function processTrial (currentTrial) {
     const statuses = ['Pending', 'Upcoming', 'In Progress', 'Completed'];
     const status = statuses[Math.floor(Math.random() * 4)];
     // TODO: Currently fake data, make this live data
-    const targetCount = Math.floor(Math.random() * 900 + 500);
+    const targetCount = moment(trial.targetCount);
```
Comment (ChristianMurphy) [inline presenter/dashboard.js]: No need to feed this through moment JS here.

**Comment by ChristianMurphy** [comment]
Other than comment.
LGTM :+1:

**Comment by sylvr3** [comment]
If I remove that line (and change that return value of that function to targetCount: trial.targetCount), it breaks the server and causes the page not to load :disappointed:

**Comment by ChristianMurphy** [comment]
It works :rainbow: :tada: 
magic

**Pull request merged**